pkg/rekor/identity.go

```diff
@@ -104,32 +107,11 @@ func (e *IdentityEntry) String() string {
 	return strings.Join(parts, " ")
 }
 
-// parseObjectIdentifier parses a string representing an ObjectIdentifier in dot notation
-// and converts it into an asn1.ObjectIdentiifer.
-func parseObjectIdentifier(oid string) (asn1.ObjectIdentifier, error) {
-	if len(oid) == 0 {
-		return nil, errors.New("could not parse object identifier: empty input")
-	}
-	nodes := strings.Split(oid, ".")
-	objectIdentifier := make([]int, len(nodes))
-	for i, node := range nodes {
-		if strings.TrimSpace(node) == "" {
-			return nil, errors.New("could not parse object identifier: no characters between two dots")
-		}
-		intNode, err := strconv.Atoi(node)
-		if err != nil {
-			return nil, err
-		}
-		objectIdentifier[i] = intNode
-	}
-	return asn1.ObjectIdentifier(objectIdentifier), nil
-}
-
 // mergeOIDMatchers groups all OID matchers from OIDMatchers, FulcioExtensions, and CustomOIDs into one slice of OIDMatchers
 func mergeOIDMatchers(mvs MonitoredValues) ([]identity.OIDMatcher, error) {
 	fulcioOIDMatchers, err := mvs.FulcioExtensions.RenderFulcioOIDMatchers()
 	if err != nil {
-		return nil, fmt.Errorf("error rendering OID Matchers from Fulcio OID extensions: %w", err)
+		return nil, fmt.Errorf("error rendering OID matchers from Fulcio OID extensions: %w", err)
 	}
 	// map of all OID extensions to all associated matching extension values
 	oidMap := make(map[string]map[string]bool)
```

Comment:
Are there any values in `mvs.OIDMatchers` when this runs? From looking over the code, I think we set `mvs.OIDMatchers` after returning from `mergeOIDMatchers`, but not before?

Reply:
if mvs.OIDMatchers was set when this was called from MatchedIndices (seems like unless MatchedIndices was specifically called by itself, since runConsistencyCheck doesn't populate OIDMatchers), otherwise it will be empty